services/common/src/redux/actions/documentActions.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable */
```

Comment:
what's eslint being disabled for? I find a lot of these get copied over but aren't necessary.

Reply:
Good question.  It's been removed.  Hadn't looked much at the file, just copied it over.  Seems to still work find without it. 🤷🏻